launch/mavros_posix_tests_iris.launch

```diff
@@ -15,7 +15,10 @@
     </include>
 
     <group ns="$(arg ns)">
+        <test test-name="box" pkg="px4" type="mission_test.py" time-limit="120.0" args="multirotor_box.mission" />
```

Comment:
I thought it would be good to add a very quick multirotor missoin since we weren't testing that before.